spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe "schema" do
+    it { should have_db_column(:email).of_type(:string).with_options(null: false) }
+    it { should have_db_column(:encrypted_password).of_type(:string) }
+
+    it { should have_db_column(:updated_at) }
+    it { should have_db_column(:created_at) }
```

Comment:
Can we stick to `expect` syntax? We also probably want to set this in the spec_helper configuration:
```ruby
RSpec.configure do |config|
  config.expect_with :rspec do |expectations|
    # Enable only the newer, non-monkey-patching expect syntax.
    # For more details, see:
    #   - http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax
    expectations.syntax = :expect
  end
end
```